src/app/components/forms/documents/UploadsByCategory.tsx

```diff
@@ -113,8 +114,8 @@ export default function UploadsByCategory({
   };
 
   const getItemIndex = (uuid: string) => {
-    const index = fileUploads.findIndex(
-      (f: FileUploadField) => f.uuid === uuid
+    const index = actesAdministratifs.findIndex(
+      (f: ActeAdministratifField) => f.uuid === uuid
```

Comment:
Puisque je n'aime pas trop les variables à une lettre ^^
```suggestion
      (acteAdministratif: ActeAdministratifField) => acteAdministratif.uuid === uuid
```